### ISSUE TITLE
Cant get league of legends on razer Synapse or get  files to work

### ISSUE THREAD

**Issue by KiaraMacD** [nicolasdeory/leagueoflegends-led#13]
Razer Synapse apps will not pickup League of Legends or the files I downloaded from here. How do I activate the files or get them to work from dowlnloading them? My Razer Chroma won't pick it up. Thanks, glad to see League in the Chroma Workshop!

**Comment by nicolasdeory** [comment]
Hi, I'm not sure I understand the issue. You mean that you have to manually open the program, it doesn't start automatically with League of Legends?

**Comment by KiaraMacD** [comment]
Thanks for the reply! I downloaded the files on my computer but there was no guide on how to get them to work on Razer Chroma? I use Razer and I contacted their support because League of Legends doesn't show up in Razer synapse apps and support told me to ask the Dev of the workshop content. I just wasn't sure if maybe I missed a step to get the League of Legends chroma display to work. Probably an issue in me not knowing how to get it to work. Thanks again!
